test/config.sh

```diff
@@ -214,5 +215,47 @@ collect_coverage() {
     mv cover.router.prof $(mktemp -u ./coverage/integration.XXXXXXXX) || true
 }
 
+################################
+# performance utils
+################################
```

Comment:
Yes that sounds like a good idea.  I've been meaning to do that for the dns tests.